### PR TITLE
completed css-brochure-form -Andrew Owyang

### PR DESCRIPTION
**Pull request by Ajowyang**
Screenshot 2024-07-17 at 10 56 05 AM

### CODE REVIEW

File: css-brochure-form/index.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    </div>
+
+    <div>
+      <form action="submit">
+        <div class="field">
+          <label for="companyEmail">COMPANY EMAIL</label>
+          <div>
+            <input type="text" value="sally@company.co" />
+          </div>
+        </div>
```
Comment (brandon-moy) [inline css-brochure-form/index.html]: Your `labels` have a `for` attribute, but where is it connecting to?

File: css-brochure-form/index.html [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+        <div class="field">
+          <label for="name">NAME</label>
+          <div>
+            <input type="text" value="Sally Smith" />
+          </div>
```
Comment (brandon-moy) [inline css-brochure-form/index.html]: Same thing here with the `label` and `for` attribute

File: css-brochure-form/index.html [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+        </div>
+
+        <div class="field">
+          <label for="whereHiring">WHERE ARE YOU HIRING?</label>
+          <div>
+            <input type="text" value="City, Country" />
+          </div>
```
Comment (brandon-moy) [inline css-brochure-form/index.html]: Same thing here with the `label` and `for` attribute

File: css-brochure-form/index.html [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+        <div class="field">
+          <div>
+            <label for="howManyEngineers"
+              >HOW MANY ENGINEERS ARE YOU HIRING IN THE NEXT 6 MONTHS?</label
+            >
+          </div>
+          <select>
+            <option>1-2 Engineers</option>
+          </select>
+        </div>
```
Comment (brandon-moy) [inline css-brochure-form/index.html]: Same thing here with the `label` and `for` attribute.

Also if it is a `select`, should it only have one `option`? Even if you don't see it I would recommend adding 2 more options here. Also, what attribute should an `option` have so we know which was selected?

**Review by brandon-moy — CHANGES_REQUESTED** [review]
Great start! Have a few changes requested:

- Label boldness
- Label connecting to inputs
- Button text capitalization

**Comment by Ajowyang** [comment]
Screenshot 2024-07-17 at 3 13 32 PM

**Review by brandon-moy — CHANGES_REQUESTED** [review]
Still need to address:

- Labels connecting to correct inputs/form elements
- Additional select options and how we get the value of the select

**Review by brandon-moy — APPROVED** [review]
👍

**Pull request merged**